nextcord/guild_preview.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional
+
+from .asset import Asset
+from .emoji import Emoji
+from .sticker import GuildSticker
+
+if TYPE_CHECKING:
+    from .state import ConnectionState
+    from .types.guild import GuildFeature, GuildPreview as GuildPreviewPayload
+
+__all__ = ("GuildPreview",)
+
+
+class GuildPreview:
+    """Represents a Discord guild preview.
+
+         .. versionadded :: 2.6
```

Comment:
```suggestion
    """Represents a Discord guild preview.

    .. versionadded :: 2.6
```
I accidentally added too much whitespace in my original suggestion, sorry about that.